### PR TITLE
troubleshooting forman

### PR DESCRIPTION
**Pull request by jvollhueter**
I think a mistake has occurred in the conversion of the units of salinity

### CODE REVIEW

File: ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def calculateBelowgroundResources(self):
             # eq. requires salinity in ppt not kg/kg
             self.belowground_resources = (1 / (1 + np.exp(
                 np.array(self._salt_effect_d) *
-                (np.array(self._salt_effect_ui) - (salinity_plant/10**3)))))
+                (np.array(self._salt_effect_ui) - (salinity_plant*10**3)))))
```
Comment (mcwimm) [inline ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.py:60]: You are right. Does this effect any of the tests or examples?

**Review by mcwimm — COMMENTED** [review]
I used this PR to tackle the issue of multiple response functions within FixedSalinity (i.e., different plant types). That's why I splitted the species file... now the file should only contain the parameterization for 1 plant module. It also contains a parameter defining the response to salinity of this plant type. In FS, the bg_factor is calculated accordingly...

With the orignal bug (/10^3 instead of *10^3) the CI test was successful. I updated all becnhmark reference files containing a combination of Kiwi and FixedSalinity.

**Review by jvollhueter — COMMENTED** [review]
Everything fine! Nice way to solve this problem! But I can not approve my own PR - so you have to do this for merging

**Review by mcwimm — APPROVED** [review]
👍

**Pull request merged**